.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
It will be great to sync it with b-c:
https://github.com/brave/brave-core/blob/master/.prettierrc.js